USAGE.md

```diff
@@ -0,0 +1,64 @@
+# Installation
+
+Please refer to the [installation instructions](README.md#setup-cli-local) for installing ksctl
+
+# Usage
+
+After you have built ksctl locally, you can use the following steps to set it up. We will use connect to a Civo account for this example, but the steps will remain the same for any cloud provider.
+
+- [Register your cloud credentials](#register-credentials)
+- [Creating a cluster](#create-a-cluster)
+- [Saving the Kubeconfig file](#saving-the-kubeconfig)
+- [Switching between clusters](#switching-between-multiple-clusters)
+- [Deleting Cluster](#deleteing-your-cluster)
+- [Creating a High Avaliability cluster](#creating-a-ha-cluster)
+
+#### Register Credentials
+
+- From your Civo dashboard, under profile > Security tab, copy your API key
+![](https://i.imgur.com/jexwOeu.png)
+- Open a terminal and type in `ksctl cred`
+ ![](https://i.imgur.com/fIWyqlH.png)
+ - Select the cloud provider you wish to register. In this case, we are using Civo, so we will type `3` and enter.
+ - Paste your API key when prompted and hit enter.
+
+
+Now, ksctl is connected to your civo account. We can now move ahead and create a cluster.
```

Comment:
Should we also add after adding api key what happens behind the scene what is fike structure created?

Reason: for a new contributor we should add content that explains some behind the scene(maybe in docs)

Reply:
As it's a usage guide we should not mentions the implementation details
Instead we can add stuff in docs folder so if later project grow we can build some website out of it